plugins/repository-gcs/qa/google-cloud-storage/build.gradle

```diff
@@ -76,6 +84,34 @@ task createServiceAccountFile() {
     }
 }
 
+task thirdPartyTest (type: Test) {
+    include '**/GoogleCloudStorageThirdPartyTests.class'
+    systemProperty 'tests.security.manager', false
+    systemProperty 'test.google.bucket', gcsBucket
+    systemProperty 'test.google.base', gcsBasePath
+    nonInputProperties.systemProperty 'test.google.account', "${ -> encodedCredentials.call() }"
+}
+
+if (useFixture) {
+    thirdPartyTest.dependsOn createServiceAccountFile, googleCloudStorageFixture
```

Comment:
Lets add a TODO here as well, this is really messy. We should just run the fixture inside the test JVM for third party tests when using the fixture. It just makes the build slower and much harder to debug not having the fixture in the same JVM. (Moreover, actually having the fixture in the same JVM would allow us to get rid of the weird and questionably valid `MockStorageImpl` we currently use.)